### ISSUE TITLE
Issue with AdamW optimizer in tensorflow 1.15.5

### ISSUE THREAD

**Issue by purvang3** [tensorflow/tensorflow#53757]
<em>Please make sure that this is an issue related to performance of TensorFlow.
As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:performance_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):  yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.15.5
- Python version: 3.6.8
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: cuda_11.5.r11.5,  nvidia docker image
- GPU model and memory: Nvidia 2080, 8 GB

You can collect some of this information using our environment capture
[script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with:
1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`
2. TF 2.0: `python -c "import tensorflow as tf; print(tf.version.GIT_VERSION, tf.version.VERSION)"`

**Describe the current behavior**

 Training object detection model with "SSD_INCEPTION_V2" as backbone. Updated AdamW optimizer from contrib module using following code in optimizer_buider.py.
    """
    config = optimizer_config.adamw_optimizer
    learning_rate = _create_learning_rate(config.learning_rate) (# learning rate type: exponential_decay_learning_rate)
    summary_vars.append(learning_rate)
    optimizer = tf.contrib.opt.AdamWOptimizer(weight_decay=4e-05, learning_rate=learning_rate)
    """

   PROBLEM: 
  When I use Adam optimizer, I can see progress in learning from coco metrics and it reaches 0.16 mAP for "0.50 IOU area ALL" 
  after 8 epochs and both training loss and validation loss decrease with small difference in values (good generalization). but when I changed to AdamW optimizer using described code, coco metric (almost 0.0) as well as loss (which is stationary) not changing at all (no generalization). 

**Describe the expected behavior**
  As per paper (https://arxiv.org/abs/1711.05101) and also from documentation, it suppose to train better and converge faster. 
 Am i missing here something? Let me know if you need any additional information.

**Standalone code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/Jupyter/any notebook.

**Other info / logs** Include any logs or source code that would be helpful to
diagnose the problem. If including tracebacks, please include the full
traceback. Large logs and files should be attached.

**Comment by tilakrayal** [comment]
@purvang3 ,
We see that you are using tf version 1.15, 1.x is not actively supported, please update to tf v2.7 and let us know if you are using same issue.Also please take a look at this [link](https://www.tensorflow.org/guide/migrate).It helps.Thanks
